lightning/src/blinded_path/message.rs

```diff
@@ -84,10 +86,61 @@ impl Writeable for ReceiveTlvs {
 	}
 }
 
+impl Readable for ReceiveTlvs {
+	fn read<R: io::Read>(r: &mut R) -> Result<Self, DecodeError> {
+		_init_and_read_tlv_stream!(r, {
+			(6, path_id, option),
```

Comment:
Spoke with @TheBlueMatt offline. Let's use a higher number like we do for payment `ReceiveTlvs` instead of using `path_id`. That way if we ever want to create blinded paths for other implementations, we can do so using `path_id`. Also, rename `path_id` to `context` in `ReceiveTlvs`.